bugbug/models/component_nn.py

```diff
@@ -178,7 +205,14 @@ def __init__(self, *args, **kwargs):
             'long_desc_vocab_size': self.long_desc_vocab_size,
             'long_desc_emb_sz': self.long_desc_emb_sz
         }
-        self.clf = ComponentNNClassifier(**kwargs)
+
+        estimators = []
+        for i, param in enumerate(self.params):
+            kwargs['params'] = param
+            estimator = ComponentNNClassifier(**kwargs)
+            estimators.append(('model_{}'.format(i), estimator))
+
+        self.clf = VotingClassifier(estimators=estimators, voting='soft', weights=[1, 1, 1])
```

Comment:
If the VotingClassifier is doing the label encoding for us, then maybe we have to remove the encoding from KerasClassifier, or make it optional.

(I wonder why some classifiers do the encoding by themselves and some don't, it's painful!)